datacenter/ucp/2.1/guides/user/services/hrm-labels.md

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
When you're adding new articles, you also need to add [an entry here](https://github.com/docker/docker.github.io/blob/master/_data/toc.yaml#L1268-L1269), to make the article show in the left navigation bar of docs.docker.com